public/javascripts/teamImage.js

```diff
@@ -0,0 +1,45 @@
+(function(){
+  //initialize the angular app and inject dependencies.
+  angular.module("olinbaja.profileimage", ['ngRoute', 'ngFileUpload'])
+    .config(function($routeProvider, $locationProvider) {
+      $routeProvider
+        .when('/uploadProfile', {
+          templateUrl: './pages/uploadProfile.html',
+          controller: 'teamImageController',
```

Comment:
Let's capitalize all uses of `teamImageController` to `TeamImageController` to be consistent with the capitalization of the rest of our controllers